writeups/quic-manageability.txt

```diff
@@ -0,0 +1,80 @@
+
+Working Group Summary:
+
+This document has been reviewed and contributed to by many participants in the QUIC WG, including those from companies with major QUIC deployments and operators of networks with significant QUIC traffic volumes.
+
+Document Quality:
+
+This document is informational and meant for network operators. There has been extensive feedback on the contents from implementers, QUIC deployments, network operators, and researchers. We believe it to be high quality information about QUIC for network operators.
+
+Personnel:
+
+Who is the Document Shepherd? Who is the Responsible Area Director?
+Matt Joras is the document shepherd, Zahed Sarker the AD. 
+
+(3) Briefly describe the review of this document that was performed by the Document Shepherd. If this version of the document is not ready for publication, please explain why the document is being forwarded to the IESG. 
+I have reviewed the document several times both as an indivudal in the WG and now as a shepherd in its current form. I believe it is ready for publication.
+
+(4) Does the document Shepherd have any concerns about the depth or breadth of the reviews that have been performed? 
+No, it has gone through several reviews and we have a wide diversity of perspectives in the feedback so far.
+
+(5) Do portions of the document need review from a particular or from broader perspective, e.g., security, operational complexity, AAA, DNS, DHCP, XML, or internationalization? If so, describe the review that took place. 
+I don't believe so.
+
+(6) Describe any specific concerns or issues that the Document Shepherd has with this document that the Responsible Area Director and/or the IESG should be aware of? For example, perhaps he or she is uncomfortable with certain parts of the document, or has concerns whether there really is a need for it. In any event, if the WG has discussed those issues and has indicated that it still wishes to advance the document, detail those concerns here. 
+No specific concerns.
+
+(7) Has each author confirmed that any and all appropriate IPR disclosures required for full conformance with the provisions of BCP 78 and BCP 79 have already been filed. If not, explain why?
+There are no IPR disclosures require for this document.
+
+(8) Has an IPR disclosure been filed that references this document? If so, summarize any WG discussion and conclusion regarding the IPR disclosures. 
+No.
+
+(9) How solid is the WG consensus behind this document? Does it represent the strong concurrence of a few individuals, with others being silent, or does the WG as a whole understand and agree with it? 
+It has received a wide diversity of review from implementers, deployers of QUIC, network operators, and researchers. I would say the consensus is fairly good though the engagement has overall been less than the original QUIC documents.
```

Comment:
We might want to state that there were two WGLC, with feedback from the first WGLC helping to strengthen consensus overall.